diff --git a/node_modules/@terrestris/react-geo/dist/Button/SimpleButton/SimpleButton.js b/node_modules/@terrestris/react-geo/dist/Button/SimpleButton/SimpleButton.js
index 6612be5..4f5efed 100644
--- a/node_modules/@terrestris/react-geo/dist/Button/SimpleButton/SimpleButton.js
+++ b/node_modules/@terrestris/react-geo/dist/Button/SimpleButton/SimpleButton.js
@@ -1,6 +1,6 @@
 import { Button, Tooltip } from 'antd';
 import * as React from 'react';
-import { CSS_PREFIX } from '../../constants';
+import { CSS_PREFIX } from '../../constants.js';
 const defaultClassName = `${CSS_PREFIX}simplebutton`;
 const SimpleButton = ({ className, type = 'primary', tooltip, tooltipPlacement, tooltipProps = {
     mouseEnterDelay: 1.5
